+euphonic/CoherentSqw.m

```diff
@@ -0,0 +1,22 @@
+classdef CoherentSqw < light_python_wrapper.light_python_wrapper
```

Comment:
Thinking about the naming, from Matlab we have `CoherentSqw`, and from Python we have `EuphonicWrapper`. For consistency should they both have the same names? And perhaps `CoherentSqw` should be named something like `CoherentCrystal` to allow for `IncoherentCrystal`, `IncoherentPowder`.. etc. in the future?

Reply:
Yes, I think it should be renamed to be consistent and `CoherentCrystal` is good.